### DISCUSSION TITLE
Readiness Feature Feedback

### DISCUSSION THREAD

**Discussion by jefftriplett** [djangopackages/djangopackages#931] [Feature Ideas]
The goal here is to collect feedback for the readiness feature I outlined in this post: 

https://mastodon.social/@webology/109892588745346437

<img width="1186" alt="Xnapper-2023-02-19-08 53 26" src="https://user-images.githubusercontent.com/50527/219967104-05280f3e-28dc-4ca9-9a29-b9a579e1d9b5.png">

**Comment by jefftriplett** [comment]
> I was looking at pyreadiness and it claims package X doesn't support say, 3.4, but at closer it look, it supports 3.4 in older versions of the package. I think people on old versions would only care if at least 1 version supports their out of date version.

[Matthew Martin's post](https://mastodon.social/@mistersql/109892698641004052) makes me realize there isn't a strong case to try and show outdated versions of Python and Django. We should avoid showing readiness features for outdated versions since we don't store the historical context.

**Reply by hugovk** [reply]
That package may have supported 3.4 once upon a time, but right now it doesn't. 

Maybe an old release still installs on 3.4, maybe not, maybe it's full of security vulnerabilities, maybe not. 

All we can safely say is that, right now, the maintainers do not support 3.4 in the latest release.

**Reply by jefftriplett** [reply]
That's a great point. I think the goal/focus of readiness should be to keep up with supported versions of Python (and Django).
